src/api/PDFDocument.ts

```diff
@@ -76,6 +79,16 @@ import { CipherTransformFactory } from '../core/crypto';
 import PDFSvg from './PDFSvg';
 import PDFSecurity, { SecurityOptions } from '../core/security/PDFSecurity';
 
+export type PDFAttachment = {
+  name: string;
+  data: Uint8Array;
+  mimeType: string | undefined;
+  afRelationship: AFRelationship | undefined;
+  description: string | undefined;
+  creationDate: Date | undefined;
+  modificationDate: Date | undefined;
+};
+
```

Comment:
The feature would also work without exporting the type. Feel free to remove that.